.github/workflows/lint.yaml

```diff
@@ -92,4 +92,7 @@ jobs:
       - name: Run black
         run: black --check --include "(tests|scripts)" .
 
+      - name: Run blackadder
+        run: blackadder --check --diff --fast --include '\.vy$' .
+
```

Comment:
Can this be a completely separate job actually? Like how `brownie` is a job, make this one into `vyper`. I know it's more set up, but it makes it logically clearer what failed in CI at quick glance

Reply:
Sure